Minecraft_BTE_Tips/main.css

```diff
@@ -0,0 +1,372 @@
+}
+.nav-link:hover {
+  color: #000;
+}
+.navbar-collapse {
+  justify-content: flex-end;
+}
+.footer_icon {
+  margin-left: auto;
+  margin-right: auto;
+  width: 3em;
+  font-size: 28px;
+  padding-top: 2px;
+  padding-bottom: 16px;
+  color: #000;
+}
+.footer_icon:hover {
+  transform: scale(1.2);
+  transition: all 0.5s;
+}
+footer {
+  margin: 0;
+  background-color: var(--nav-bar-black);
+  color: rgba(26, 24, 24, 0.8);
+  text-align: center;
+  padding: 1.85em;
+}
+.heading,
+.info,
+.btn,
+.images,
+.mcbuild_pic,
+.mcbuild_info {
+  margin-left: 5px;
+}
+
+.fa-discord {
+  color: #7289da;
+}
+
+.fa-instagram {
+  color: #e1306c;
+}
+
+.fa-github {
+  color: #24292e;
+}
+
+.fa-youtube {
+  color: #ff0000;
+}
+
+
+.fa-twitter {
+  color: #1da1f2;
+}
+
+.slider {
+  position: relative;
+  background: #000116;
+  width: 800px;
+  min-height: 500px;
+  margin: auto;
+  overflow: hidden;
+  border-radius: 10px;
+}
+
+.slider .slide {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  clip-path: circle(0% at 0 50%);
+}
+
+.slider .slide.active {
+  clip-path: circle(150% at 0 50%);
+  transition: 2s;
+}
+
+.slider .slide img {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.slider .slide .info {
+  position: absolute;
+  color: #222;
+  background: rgba(255, 255, 255, 0.3);
+  width: 75%;
+  margin-top: 50px;
+  margin-left: 50px;
+  padding: 20px;
+  border-radius: 5px;
+  box-shadow: 0 5px 25px rgb(1 1 1 / 5%);
+}
+
+.slider .slide .info h2 {
+  font-size: 2em;
+  font-weight: 800;
+}
+
+.slider .slide .info p {
+  font-size: 1em;
+  font-weight: 400;
+}
+
+.navigation {
+  height: 500px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  opacity: 0;
+  transition: opacity 0.5s ease;
+}
+
+.slider:hover .navigation {
+  opacity: 1;
+}
+
+.prev-btn,
+.next-btn {
+  z-index: 999;
+  font-size: 2em;
+  color: #222;
+  background: rgba(255, 255, 255, 0.8);
+  padding: 10px;
+  cursor: pointer;
+}
+
+.prev-btn {
+  border-top-right-radius: 3px;
+  border-bottom-right-radius: 3px;
+}
+
+.next-btn {
+  border-top-left-radius: 3px;
+  border-bottom-left-radius: 3px;
+}
+
+.navigation-visibility {
+  z-index: 999;
+  display: flex;
+  justify-content: center;
+}
+
+.navigation-visibility .slide-icon {
+  z-index: 999;
+  background: rgba(255, 255, 255, 0.5);
+  width: 20px;
+  height: 10px;
+  transform: translateY(-50px);
+  margin: 0 6px;
+  border-radius: 2px;
+  box-shadow: 0 5px 25px rgb(1 1 1 / 20%);
+}
+
+.navigation-visibility .slide-icon.active {
+  background: #4285f4;
+}
+
+@media (max-width: 900px) {
+  .slider {
+    width: 100%;
+    margin-left: 3px;
+  }
+
+  .slider .slide .info {
+    position: relative;
+    width: 80%;
+  }
+}
+
+@media (max-width: 500px) {
+  .slider .slide .info h2 {
+    font-size: 1.8em;
+    line-height: 40px;
+    margin-left: 2px;
+    margin-right: 8px;
+  }
+
+  .slider .slide .info p {
+    font-size: 0.9em;
+  }
+}
+
```

Comment:
This the CSS for the image slideshow on the landing page, so there is no issue of these over here